<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCMsgSocketDialog</class>
 <widget class="QDialog" name="PCMsgSocketDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>368</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Scoket 双向通信</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="Label_DestIP">
       <property name="text">
        <string>目的IP(&amp;I)：</string>
       </property>
       <property name="buddy">
        <cstring>LineEdit_DestIP</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="LineEdit_DestIP"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Label_DestPort">
       <property name="text">
        <string>目的端口(&amp;P)：</string>
       </property>
       <property name="buddy">
        <cstring>LineEdit_DestPort</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="LineEdit_DestPort"/>
       </item>
       <item>
        <widget class="QPushButton" name="Button_Connect">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>连接</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="List_Text"/>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="Label_SrcIP">
       <property name="text">
        <string>本机IP：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="LineEdit_SrcIP">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Local_SrcPort">
       <property name="text">
        <string>本机端口(&amp;S)：</string>
       </property>
       <property name="buddy">
        <cstring>LineEdit_SrcPort</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="LineEdit_SrcPort"/>
       </item>
       <item>
        <widget class="QPushButton" name="Button_Listen">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>监听</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="LineEdit_SendMsg"/>
     </item>
     <item>
      <widget class="QPushButton" name="Button_SendMsg">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Button_Close">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>LineEdit_DestIP</zorder>
  <zorder>Button_Connect</zorder>
  <zorder>Label_DestIP</zorder>
  <zorder>Label_DestPort</zorder>
  <zorder>LineEdit_DestPort</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>List_Text</zorder>
  <zorder>line</zorder>
  <zorder>Label_SrcIP</zorder>
  <zorder>Local_SrcPort</zorder>
  <zorder>LineEdit_SrcIP</zorder>
  <zorder>LineEdit_SrcPort</zorder>
  <zorder>Button_SendMsg</zorder>
  <zorder>LineEdit_SendMsg</zorder>
  <zorder>Local_SrcPort</zorder>
  <zorder>LineEdit_DestPort</zorder>
  <zorder>Button_Connect</zorder>
  <zorder>Button_Listen</zorder>
  <zorder>LineEdit_SrcPort</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>LineEdit_DestIP</tabstop>
  <tabstop>List_Text</tabstop>
  <tabstop>LineEdit_SrcIP</tabstop>
  <tabstop>Button_Close</tabstop>
 </tabstops>
 <resources>
  <include location="pcmsgsocket.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Button_Close</sender>
   <signal>clicked()</signal>
   <receiver>PCMsgSocketDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
